ChaosMod/Effects/db/Misc/MiscNoWater.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+	Effect by MoneyWasted
+	Special Thanks to Parik for the Help :)
+*/
+
+#include <stdafx.h>
+
+struct CWaterQuad
+{
+	short MinX;   // 0x0
+	short MinY;   // 0x2
+	short MaxX;   // 0x4
+	short MaxY;   // 0x6
+	uint Color;   // 0x8
+	char unk1[4]; // 0xC
+	char unk2[4]; // 0x10
+	float Z;      // 0x14
+	uint Flags;   // 0x18
+};
+static_assert(sizeof(CWaterQuad) == 0x1C);
+
+static CWaterQuad* WaterQuads;
+static std::vector<float> WaterHeights;
+
+static CWaterQuad* GetWaterQuads()
+{
+	Handle handle = Memory::FindPattern("? 6B C9 1C ? 03 0D ? ? ? ? 66 ? 03 C5 66 89 05 ? ? ? ?");
+	if (handle.IsValid())
+	{
+		return *handle.At(6).Into().Get<CWaterQuad*>();
+	}
+}
+
+static void OnStart()
+{
+	WaterQuads = GetWaterQuads();
```

Comment:
Would be great to have a null check here (and for OnStop too) in case the pattern wasn't found so rather than the mod crashing the effect does nothing